templates/tgi_h100.template.slurm

```diff
@@ -45,7 +45,7 @@ srun --container-image='ghcr.io#huggingface/text-generation-inference' \
     --revision $revision \
     --max-concurrent-requests 2000 \
     --max-total-tokens 8192 \
-    --max-input-length 7144 \
+    --max-input-length 2048 \
```

Comment:
Oh this might cause problems for constitutional AI as it has very long few shot examples. Could we reverse it back? Alternatively we could make this configurable like the following and make the template ` --max-input-length {{tgi_max_input_length}}`. WDYT?

```python benchmark.py --tgi-max-input-length=2048```

Reply:
Ah sorry, I thought your prompts were short too, 7144 is fine I'll revert the chnage (I post-processed my prompts now so most of them are short and no need to force TGI to truncate)